assets/website/lang/webmessages_pl.properties

```diff
@@ -82,9 +82,9 @@ modal_body = "Jeżeli nie pamiętasz danych logowania swojego starego konta - ni
 modal_button_register = "Zarejestruj nowe konto"
 
 [dashboard]
-name = "Dashboard"
+name = "Kokpit"
```

Comment:
Dashboard is rarely translated into polish afaik

Reply:
But I don't think I should leave it untranslated.